scripts/cmake/MCWAMP.cmake

```diff
@@ -78,29 +94,13 @@ macro(add_mcwamp_library_hc_am name )
   # add HSA libraries
   target_link_libraries(${name} ${HSA_LIBRARY})
   target_link_libraries(${name} pthread)
-  target_link_libraries(${name} c++)
-  target_link_libraries(${name} c++abi)
-endmacro(add_mcwamp_library_hc_am name )
 
-####################
-# C++AMP config (clamp-config)
-####################
+  if (USE_LIBCXX)
+    target_link_libraries(${name} c++)
+    target_link_libraries(${name} c++abi)
+  endif (USE_LIBCXX)
 
-macro(add_mcwamp_executable name )
```

Comment:
This is not needed anymore. hcc-config has its own cmakefile hcc_config/CMakeLists.txt that doesn't depend on this macro